### PR TITLE
Making operations test package parallell 

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description
Made operations package independent, to make it parallel run with other packages.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - NA
2. Unit tests - NA
3. Integration tests - Automated

### CODE REVIEW

File: tools/integration_tests/operations/copy_dir_test.go [hunk 1]
```diff
@@ -242,17 +241,16 @@ func checkIfCopiedEmptyDirectoryHasNoData(destSrc string, t *testing.T) {
...
+	srcDir := path.Join(testDir, EmptySrcDirectoryCopyTest)
 	operations.CreateDirectoryWithNFiles(0, srcDir, "", t)
 
 	// Create below directory
 	// destNonEmptyCopyDirectory                                                -- Dir
 	// destNonEmptyCopyDirectory/subDirInNonEmptyDestCopyDirectory              -- Dir
-	destDir := path.Join(setup.MntDir(), DestNonEmptyCopyDirectory)
-	createDestNonEmptyDirectory(t)
+	destDir := path.Join(testDir, DestNonEmptyCopyDirectory)
+	createDestNonEmptyDirectory(destDir, t)
```
Comment (ashmeenkaur) [inline tools/integration_tests/operations/copy_dir_test.go:253]: Nit, can we merge line 252 and 253 into the same function. You can return dirPath from the create call for ease of readability

File: tools/integration_tests/util/setup/setup.go [hunk 2]
```diff
@@ -146,11 +146,11 @@ func CompareFileContents(t *testing.T, fileName string, fileContent string) {
 	}
 }
 
-func CreateTempFile() string {
+func CreateTempFile(dirPath string) string {
```
Comment (ashmeenkaur) [inline tools/integration_tests/util/setup/setup.go:149]: Nit: There is a similar method in file_operations.go, shall we use it instead? I also think that we should pass filepath to this method as passing dirName is not very intuitive.

File: tools/integration_tests/operations/write_test.go [hunk 3]
```diff
@@ -64,14 +65,14 @@ func validateExtendedObjectAttributesNonEmpty(objectName string, t *testing.T) *
 }
 
 func validateObjectAttributes(attr1, attr2 *storage.ObjectAttrs, t *testing.T) {
-	const contentType = "text/plain; charset=utf-8"
+	const ContentType = "text/plain; charset=utf-8"
```
Comment (ashmeenkaur) [inline tools/integration_tests/operations/write_test.go:68]: Why was this change required?

Comment (Tulsishah) [reply tools/integration_tests/operations/write_test.go]: Accidentally introduced. Fixed

File: tools/integration_tests/operations/copy_dir_test.go [hunk 4]
```diff
@@ -29,23 +29,20 @@ import (
...
-func createSrcDirectoryWithObjects(dirPath string, t *testing.T) {
-	// Clean the mountedDirectory before running test.
-	setup.CleanMntDir()
-
+func createSrcDirectoryWithObjects(dirPath string, t *testing.T) string {
 	// testBucket/srcCopyDir
 	err := os.Mkdir(dirPath, setup.FilePermission_0600)
 	if err != nil {
 		t.Errorf("Mkdir at %q: %v", dirPath, err)
-		return
```
Comment (ashmeenkaur) [inline tools/integration_tests/operations/copy_dir_test.go:40]: h

File: tools/integration_tests/operations/copy_dir_test.go [hunk 5]
```diff
@@ -29,23 +29,20 @@ import (
 // srcCopyDir               -- Dir
 // srcCopyDir/copy.txt      -- File
 // srcCopyDir/subSrcCopyDir -- Dir
-func createSrcDirectoryWithObjects(dirPath string, t *testing.T) {
```
Comment (ashmeenkaur) [inline tools/integration_tests/operations/copy_dir_test.go:32]: h

**Review by ashmeenkaur — CHANGES_REQUESTED** [review]

**Review by ashmeenkaur — APPROVED** [review]

**Pull request merged**